### PR TITLE
Bug SOL-119911:  Upgrade solution file to VS2015

### PR DESCRIPTION
**Pull request by RagnarPaulson**
Upgrade solutions file provided for Windows to VS2015.
- update README
- update .bat scripts appropriately.
- update all project files
- set default/activecfg to x64/Release instead of x86/Debug

### CODE REVIEW

File: build/intro/win/VS2015/Debug/BasicReplier.pdb
Comment (cjwmorgan-sol) [inline build/intro/win/VS2015/Debug/BasicReplier.pdb:1]: This is for all files and the folder Debug. Looks like build files please remove, maybe add a git ignore for even?

File: build/intro/win/VS2015/.vs/Intro/v14/.suo
Comment (cjwmorgan-sol) [inline build/intro/win/VS2015/.vs/Intro/v14/.suo:1]: Is this file needed or even the .vs folders?

**Review by cjwmorgan-sol — COMMENTED** [review]
Looks like there are some build files pushed in. Maybe we need a gitignore for these files?

File: build/intro/win/VS2015/Debug/BasicReplier.pdb
Comment (cjwmorgan-sol) [reply build/intro/win/VS2015/Debug/BasicReplier.pdb:1]: Got removed.

File: build/intro/win/VS2015/.vs/Intro/v14/.suo
Comment (cjwmorgan-sol) [reply build/intro/win/VS2015/.vs/Intro/v14/.suo:1]: Got removed

**Review by cjwmorgan-sol — APPROVED** [review]
Looks good to me.
I could build Debug and release x64 using vs2022 locally as well.

**Comment by aaron-613** [comment]
Thanks for the review, @cjwmorgan-sol .  Merging.
